.golangci.yml

```diff
@@ -1,9 +1,6 @@
 service:
-  golangci-lint-version: 1.15.x # use the fixed version to not introduce new linters unexpectedly
+  golangci-lint-version: 1.16.x # use the fixed version to not introduce new linters unexpectedly
```

Comment:
/hold
You need to update bin/linters to install the right version.

Reply:
Please add a comment as well for the next person to keep them in sync.

Reply:
Done.